user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
+                break_loop = 0
+                print("what movie would you like to add?")
+                movie_id = movie_funcs.browse_movies(conn)
+                if movie_id == -1:
+                    print("Was not able to get movieID to add to collection.\nWill Exit...")
+                    curs.execute("DELETE FROM moviecollection WHERE name = %s AND madeby = %s", (collection_Name, user_ID,))
+                    return
+                else:
+                    movie_ID = str(movie_id)
+                    curs.execute(add_movie_collection_query, (movie_ID, collection_ID))
+                break_loop = int(input_utils.get_input_matching("\n0 - Continue adding movies\n1 - Stop adding movies: "))
+                if break_loop == 0:
+                    pass
+                elif break_loop == 1:
+                    break
+                else:
+                    while True:
+                        print(f"Invalid Input of {break_loop}, please enter a valid input")
+                        break_loop = int(input_utils.get_input_matching("""\n0 - Continue adding movies\n1 - Stop adding movies: """))
+                        if break_loop == "0" or "1":
+                            break
+                if break_loop == 0:
+                    pass
+                else:
+                    break
+            else:
+                print("Not an acceptable answer for decision! Ending function")
+                return
+    conn.commit()
+    print(f"Finished create_collection function!!")
+
+
+
+def modify_collection(conn, user_ID) -> None:
+    """
+    modify a current collection that is in the incollection table 
+
+    :param conn: The connection to the database
+    :param user_ID: The ID of the user
+    :param movie_ID: The ID of the current movie
+    :return: Nothing
+    """
+    # A Query to get the collection ID for a certain User ID
+    query = """
+    SELECT collectionid FROM moviecollection AS mc
+    WHERE mc.madeby = %s
+    ORDER BY collectionid ASC
+    """
+    # A query to remove a movie from a collection
+    remove_movie_query = """
+    DELETE FROM incollection 
+    WHERE movieid = %s AND collectionid = %s
+    """
+    # A query to add a movie into a collection
+    add_movie_query = """
+    INSERT INTO incollection (movieid, collectionid)
+    VALUES (%s, %s)
+    """
+    # Deletes a collection from the table
+    delete_moviecollection_query = """
+    DELETE FROM moviecollection 
+    WHERE collectionid = %s
+    """
+    # Deletes all movie IDs associated with the collection ID
+    delete_all_movie_query = """
+    DELETE FROM incollection
+    WHERE collectionid = %s
+    """
+
+    # Changes the name of the collection
+    change_name_query = """
+    UPDATE moviecollection
+    SET name = %s
+    WHERE collectionid = %s
+    """
+
+    
+    collection_ID = ""
+    movie_ID = movie_funcs.browse_movies(conn)
```

Comment:
You may only want to prompt the user to browse movies when selecting a movie to add. Why should the user have to select a movie to change the name of the collection?

Reply:
Done